appendices/package-approval-template.md

```diff
@@ -0,0 +1,24 @@
+```markdown
+----------------------------------------------
+🎉 <package-name-here> has been approved by pyOpenSci! Thank you <maintainer-name-here> for submitting <package-name> and many thanks to <reviewer-names-here> for reviewing this package! 😸  
+
+There are a few things left to do to wrap up this submission:
+- [ ] Activate Zenodo watching the repo if you haven't already done so.
+- [ ] Tag and create a release to create a Zenodo version and DOI.
```

Comment:
Silly question: is Zenodo a must have or maintainers are allowed to register their code on the DOI provider their prefer (or they are allowed too, in case some institutions might have strict rules about code publication)?

Reply:
Not silly at all. We definitely don't need anyone to use zenodo - any digital identifier is ok. We think it is important that the DOI provider supports versioning as the package releases update. I"m curious what other types of providers you have in mind? I could add a note about this in our packaging guide. eventually we can link many of these checks to that guide (when it's more developed).

Reply:
for the time being i've added this

*Note: it is ok if you need to use another DOI provided for your packages's release.*

Reply:
> I"m curious what other types of providers you have in mind?

I don't know a lot of people using something else... :)
I found this (quite old) [blogpost](https://blog.datacite.org/doi-registrations-software/) with a list of providers.
I know that some institutions (especially in Europe) suggests their employees to use local instances of gitlab (still lacking connection with zenodo). Some datacenters offer a DOI service and universities recommend to use the internal services (even if zenodo is CERN).
I think the note you added is enough to reassure people that pyOpenSci is not not forcing them to choose a specific platform ... just to let young PhD know that they don't need to start a conflict with hierarchy if they want to submit ... 😉
The fact that the review process is on github is not an issue thanks to the magic of git mirroring.

Reply:
ok wonderful! yup even without direct integration, you can actually still upload to zenodo manually if you wish. but figshare or any other tool that supports versioned DOI's is really fine! i'm super glad that you brought this up. thank you! i just hadn't considered it.

I know that for some the licensing is also an issue. Some places - i think industry and maybe some agencies have rules around licensing that can sometimes conflict too.

Reply:
i'll resolve this for now but feel free to ping me in slack with any other thoughts if they come up!